cmd/binfmt/main.go

```diff
@@ -133,13 +133,33 @@ func parseArch(in string) (out []string) {
 	return
 }
 
+func parseUninstall(in string) (out []string) {
+	if in == "" {
+		return
+	}
+	for _, v := range strings.Split(in, ",") {
+		if c, ok := configs[v]; ok {
```

Comment:
this should be after `platforms.Parse`

Reply:
`p.Architecture` should only be used for matching config(or just as a secondary option). Atm `aarch64` input would not work because it also parses as an arch.

Reply:
```shell
$ docker run --rm --privileged tonistiigi/binfmt:test --install arm64
2021/04/22 09:13:30 installing: arm64 OK
{
  "supported": [
    "linux/amd64",
    "linux/arm64",
    "linux/386"
  ],
  "emulators": [
    "WSLInterop",
    "qemu-aarch64"
  ]
}
```

Works fine with `aarch64`

```shell
$ docker run --rm --privileged tonistiigi/binfmt:test --uninstall aarch64
2021/04/22 09:13:36 uninstalling: qemu-aarch64 OK
{
  "supported": [
    "linux/amd64",
    "linux/386"
  ],
  "emulators": [
    "WSLInterop"
  ]
}
```


```shell
$ docker run --rm --privileged tonistiigi/binfmt:test --uninstall arm64
2021/04/22 09:15:52 uninstalling: qemu-aarch64 OK
{
  "supported": [
    "linux/amd64",
    "linux/386"
  ],
  "emulators": [
    "WSLInterop"
  ]
}
```

Reply:
But what about when the installed emulator name is just `aarch64`? Eg. installed manually or by another tool.

Reply:
Yes indeed I had not thought of this alternative